server/src/main/scala/de/frosner/broccoli/templates/TemplateRenderer.scala

```diff
@@ -0,0 +1,45 @@
+package de.frosner.broccoli.templates
+
+import javax.inject.Inject
+
+import de.frosner.broccoli.instances.conf.InstanceConfiguration
+import de.frosner.broccoli.models.{Instance, ParameterInfo, ParameterType}
+import play.api.libs.json.{JsString, JsValue, Json}
+
+/**
+  * Renders json representation of the passed instance
+  *
+  * @param instanceConfiguration
+  */
+class TemplateRenderer @Inject()(instanceConfiguration: InstanceConfiguration) {
+  private[templates] def sanitize(parameter: String, value: String, parameterInfos: Map[String, ParameterInfo]) = {
```

Comment:
Because it is a function that it is only used by `renderJson` and not supposed to be public, but I still want to unit test it.